src/web/views/check.njk

```diff
@@ -1,85 +1,74 @@
 {% extends "templates/form.njk" %}
 {% from "input/macro.njk" import govukInput %}
 {% from "table/macro.njk" import govukTable %}
+{% from "fieldset/macro.njk" import govukFieldset %}
 
-{% set buttonText = "Confirm" %}
+{% set buttonText = buttonText %}
 {% block formContent %}
+  {% call govukFieldset({
+    legend: {
+      text: heading,
+      classes: "govuk-fieldset__legend--xl",
+      isPageHeading: true
+    }
+  }) %}
+
+  {% endcall %}
   {{ govukTable({
-      caption: "Check details",
       firstCellIsHeader: true,
-      head: [
-        {
-          text: "Field"
-        },
-        {
-          text: "Value"
-        }
-      ],
       rows: [
         [
           {
-            text: "First name"
-          },
-          {
-            text: claim.firstName
-          }
-        ],
-        [
-          {
-            text: "Last name"
+            text: name
           },
           {
-            text: claim.lastName
+            text:  [claim.firstName, claim.lastName] | join(' ')
           }
         ],
         [
           {
-            text: "National Insurance number"
+            text: nino
           },
           {
             text: claim.nino
           }
         ],
         [
           {
-            text: "Date of birth"
+            text: dateOfBirth
           },
           {
             text: [claim['dateOfBirth-day'], claim['dateOfBirth-month'], claim['dateOfBirth-year']] | join(' ')
           }
         ],
         [
           {
-            text: "Are you pregnant?"
+            text: areYouPregnant
           },
           {
             text: claim.areYouPregnant
           }
         ],
+        expectedDeliveryDateRow,
```

Comment:
I'm unsure about having a single row that is built on the server - would it be better to build the complete `rows` array on the server?

The `pageContent()` function now seems like its performing two functions:
1. providing text strings to the template
2. creating structure for rendering HTML

In my mind these are two distinct functions.

Reply:
Ok, I'll build them all on the server, I'll work on it now, might be Monday before I push that commit.